packages/compass/src/main/handle-unhandled-rejection.ts

```diff
@@ -0,0 +1,48 @@
+import { app, dialog, clipboard } from 'electron';
+import cleanStack from 'clean-stack';
+import ensureError from 'ensure-error';
+import COMPASS_ICON from './icon';
+
+async function handleUnhandledRejection(err: Error): Promise<void> {
```

Comment:
Right now this is just a copy of the uncaught exception handler with literally only the console.error line changed.